### PR TITLE
Implement project schema and api

### LINKED ISSUE #8: Implement schema and REST API for the Project model.
- Write a DDL (Data Definition Language) that has the project's schema's definition.
- Write a REST API that does CRUD (Cread Read Update Delete) operations on the project model.
- Project model [reference](https://github.com/GDSC-ASU/website/blob/main/src/lib/models/Project.ts)

### PR DESCRIPTION
**Pull request by Dimah02**

### CODE REVIEW

File: src/lib/data/index.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { Sequelize } from "sequelize-typescript";
 import { Event } from "$lib/data/Event";
```
Comment (mbaraa) [inline src/lib/data/index.ts]: update this import path to `./Event`

File: src/lib/utils/requests/ProjectRequests.ts [hunk 2]
```diff
@@ -1,38 +1,28 @@
...
-		altImagePath: "/images/logogen.png",
-		link: "https://google.com",
-	},
-];
-
-export default class ProjectRequests {
-	public static async getProjects(): Promise<Project[]> {
-		return projects;
+export default class ProjectRequest {
+	public static async getAllProjects() {
```
Comment (mbaraa) [inline src/lib/utils/requests/ProjectRequests.ts:4]: reflect this renaming in `src/routes/+page.server.ts`

**Review by mbaraa — CHANGES_REQUESTED** [review]
update the changes as requested in the comments.

**Comment by Dimah02** [comment]
I updated the names of the functions in the components, but I couldn't resolve the issue on my local repo this still occurs when `npm run dev`

```
TypeError: Cannot read properties of undefined (reading 'define')
    at C:/Users/deema/Desktop/svelt/website/src/lib/data/Event.ts:4:32
```
and I noticed that the name of the database file in the data folder is gdsc.db but in the `.env.example`  is gdscasu.db, is that ok?

**Comment by mbaraa** [comment]
> I updated the names of the functions in the components, but I couldn't resolve the issue on my local repo this still occurs when `npm run dev`
> 
> ```
> TypeError: Cannot read properties of undefined (reading 'define')
>     at C:/Users/deema/Desktop/svelt/website/src/lib/data/Event.ts:4:32
> ```
> 
> and I noticed that the name of the database file in the data folder is gdsc.db but in the `.env.example` is gdscasu.db, is that ok?

looks like it's a TS import conflict issue, I'll merge your PR and I'll fix this in another PR.

**Review by mbaraa — APPROVED** [review]
LGTM

**Pull request merged**